Include/Types/Vector4.hpp

```diff
@@ -237,6 +237,11 @@ class Vector4Base
 	}
 };
 
+template<typename T>
+std::ostream& operator<<(std::ostream& os, const Vector4Base<T>& v) {
+	return os << v.x << "," << v.y << "," << v.z << "," << v.w;
+}
+
```

Comment:
I like this. I'm thinking of use cases for this and most of them would be for simple presentation or saving to a file.
So I'm wondering if it should look more python like, instead of outputting 5,5,5,5 it could have brackets around it [5,5,5,5] instead, although those brackets could easily be added by the user application as well.

Just throwing out ideas at this point, feel free to share yours.

Reply:
I don't really care how its formatted, as long as i can see the numbers for debug output. For saving to file i would use binary format for my use cases.

Reply:
I'll use brackets then.
I was thinking if it's part of a list and someone implements ostream for a list, without brackets you couldn't tell where the next element starts and ends.